.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[ci skip]')"
     strategy:
       matrix:
-        node-version: [14.17]
+        node-version: [18.4.0]
```

Comment:
Even more tests hang without this update, because when installing some libs warn of a bad (outdated) node engine.